packages/bot/src/stores/index.js

```diff
@@ -1,5 +1,9 @@
 import BotStore from './bot-store';
 
 export default class RootStore {
-    bot = new BotStore();
+    constructor(core, ws) {
+        this.bot = new BotStore(ws);
```

Comment:
@aaron-binary do we need to pass down `WS` to all store or should we get it from `rootstrore`?
 something like `get ws() { return this.rootstrore.ws; }`

Reply:
another way of doing it is we can extend all the store from `RootStore`, like `class BotStore extend RootStore` so that every store have access to all the property of `RootStore`

Reply:
@sam-binary we already have access to rootstore in all our stores cause we inject it using mobx provider